pkg/cloud/services/networking/network.go

```diff
@@ -55,6 +55,8 @@ func (s *Service) ReconcileNetwork(clusterName string, openStackCluster *infrav1
 			ID:   res.ID,
 			Name: res.Name,
 		}
+		sInfo := fmt.Sprintf("Reuse Existing Network %s with id %s", res.Name, res.ID)
+		s.logger.V(5).Info(sInfo)
```

Comment:
Do you have a reason level 5? 3,4, and 6 seems popular in this project. 5 seems new.

Reply:
nope，if you like, I can change to 4 :)

Reply:
I do not know the criteria actually :)
Sorry one more question. one line seems popular. I do not have the criteria, too.

```go
s.logger.V(5).Info("Reuse Existing Network %s with id %s", res.Name, res.ID)
```

Reply:
s.logger.V(5).Info("Reuse Existing Network %s with id %s", res.Name, res.ID) 
doesn't work, it will not format the data, I didn't find a `Infof` way like other go repo in our repo

Reply:
Sorry,  like https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/pkg/cloud/services/networking/securitygroups.go#L341,

```go
s.logger.V(5).Info("Reuse Existing Network", "name",  res.Name, "id", res.ID)
```

Reply:
yes, I knew that works , but no big difference, correct?

Reply:
yeah, so I asked.